docs/types/mdx-js.d.ts

```diff
@@ -0,0 +1,43 @@
+/**
```

Comment:
I would rather go with this one 😉 
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/mdx-js__react/index.d.ts

Reply:
https://www.npmjs.com/package/@types/mdx-js__react

Reply:
Gosh, I really can't trust their docs 😑 it's the second time their docs mismatch actual state. Thanks for checking that